management/backup.py

```diff
@@ -446,46 +446,31 @@ def list_target_files(config):
 			raise ValueError("Connection to rsync host failed: {}".format(reason))
 
 	elif target.scheme == "s3":
-		# match to a Region
-		import boto.s3
-		from boto.exception import BotoServerError
-		custom_region = False
-		for region in boto.s3.regions():
-			if region.endpoint == target.hostname:
-				break
-		else:
-			# If region is not found this is a custom region
-			custom_region = True
-
-		bucket = target.path[1:].split('/')[0]
-		path = '/'.join(target.path[1:].split('/')[1:]) + '/'
-
-		# Create a custom region with custom endpoint
-		if custom_region:
-			from boto.s3.connection import S3Connection
-			region = boto.s3.S3RegionInfo(name=bucket, endpoint=target.hostname, connection_cls=S3Connection)
+		import boto3.s3
+		from botocore.exceptions import ClientError
+		
+		# separate bucket from path in target
+		bucket = target.path.split('/')[1]
+		path = '/'.join(target.path.split('/')[2:]) + '/'
```

Comment:
This doesn't look related to boto3. Can you revert it?

Reply:
yes